src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/AggregateCrossTargetProjectContextProvider.cs

```diff
@@ -201,12 +201,11 @@ private async Task<AggregateCrossTargetProjectContext> CreateProjectContextAsync
             ImmutableDictionary<ITargetFramework, ITargetedProjectContext>.Builder innerProjectContextsBuilder = ImmutableDictionary.CreateBuilder<ITargetFramework, ITargetedProjectContext>();
             ITargetFramework activeTargetFramework = TargetFramework.Empty;
 
-            foreach (KeyValuePair<string, ConfiguredProject> kvp in configuredProjectsMap)
+            foreach ((string tfm, ConfiguredProject configuredProject) in configuredProjectsMap)
```

Comment:
How does this work? Where is the deconstruction method coming from?

Reply:
KeyValuePair [already has a](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.keyvaluepair-2.deconstruct?view=netcore-2.1&viewFallbackFrom=netframework-4.7.2) deconstruction method in .NET Framework

Reply:
That's only .NET Core 2.0/2.1. I can't see that in .NET Framework.

Reply:
Looks like Drew added it in his second week: https://github.com/dotnet/project-system/blob/29b4f9c4f2378aa9b20a1b6e269f4ca177f0fc45/src/Microsoft.VisualStudio.ProjectSystem.Managed/Composition/DeconstructionExtensions.cs#L7

The long con...

Reply:
Ha, I forgot adding that.

Reply:
Ah that makes sense.